test/specs/reference/tests/06.permissions.js

```diff
@@ -1,8 +1,44 @@
-var nock = require("nock");
+var ermrestImport = require(process.env.PWD + '/test/utils/ermrest-import.js');
```

Comment:
I think it's easier if I ask these questions in person, and If you can walk me through these test cases. But I'm writing my questions here just for a reference:

- Why are you changing ACLs for in `afterAll` and `beforeAll` for old test cases? Aren't those test cases only working based on annotation?
- This test spec is difficult to read. Can you break it into multiple specs? or at least have a top level `describe` that differentiates the test cases that are only based on annotation, and the test cases that are based on new ACLs?
- You've used arrow functions in this file which is ES6, I have nothing against it. But we should discuss about it in the group. It's odd that one file is using ES6.

Reply:
We need to change the permissions as the restrictedUser doesn't has permissions to do anything. So when we call `canCreate` the permissions are also checked with the annotations, thats the reason I had to change them. Moreover I didn't want to test it in context of admin user. Admin already has all the rights.

Secondly,  I used the arrow functions only in test-specs. These tests are run in `node.js`. We should gradually change all test specs as we make changes to them to make development easier.